lib/smart_proxy_pulp_plugin/pulp3_plugin.rb

```diff
@@ -0,0 +1,20 @@
+module PulpProxy
+  class Pulp3Plugin < ::Proxy::Plugin
+    plugin "pulp3", ::PulpProxy::VERSION
+    default_settings :pulp_url => 'https://localhost/pulp/',
+                     :pulp_dir => '/var/lib/pulp',
+                     :pulp_content_dir => '/var/lib/pulp/content',
+                     :mirror => false
+
+    expose_setting :pulp_url
+    expose_setting :mirror
+    capability( lambda do ||
+      begin
+        Pulp3Client.capabilities
+      rescue => e
```

Comment:
My preference would be to have this rescue in the client `capabilities` method - any reason to do it here?

Reply:
makes sense to me!